site/app/libraries/database/DatabaseQueries.php

```diff
@@ -818,6 +818,16 @@ public function isStaffPost($author_id) {
         return intval($this->course_db->rows()[0]['user_group']) <= 3;
     }
 
+    public function isLimitedAccessGraderPost($author_id) {
+        $this->course_db->query("SELECT user_group FROM users WHERE user_id=?", [$author_id]);
+        return intval($this->course_db->rows()[0]['user_group']) == 3;
```

Comment:
```suggestion
        return intval($this->course_db->row()['user_group']) === 3;
```

Reply:
It would also probably be good to replace the number 3 with a constant which indicates what it means.